### ISSUE TITLE
Tab1, Tab2, & Tab3 are not keyboard accessible.

### ISSUE THREAD

**Issue by msftedad** [Semantic-Org/Semantic-UI#7091]
### Title (Put in field above)
[Tab1, Tab2 & Tab3] Should be keyboard accessible.

### Description
All the controls should be keyboard accessible and focus should be clearly visible.

### Steps
1. Navigate to this link https://codesandbox.io/s/zlync7?module=/example.js
2. Tab navigate to the Tab1, Tab2, & Tab3 controls.
3. Observe that Tab1, Tab2, & Tab3 controls are keyboard accessible or no.

### Expected Result
Tab1, Tab2, and Tab3 controls should be keyboard accessible and keyboard focus should be clearly visible on the controls.

### Actual Result
Tab1, Tab2, & Tab3 controls are not keyboard accessible.

### Version
OS: 21H2 Version (22000.978)

### Attachment
https://user-images.githubusercontent.com/93735775/194360876-402c04a3-891e-4b8f-8e04-c0a3ee6f6550.mp4
